webapp/client/src/setupProxy.js

```diff
@@ -14,7 +14,7 @@ function setupProxy(app) {
         "!/*.hot-update.json", // HMR requests
       ],
       {
-        target: "http://localhost:5000",
+        target: "http://127.0.0.1:5000",
```

Comment:
For me it works with both. So, it might be some local configuration change on your side 🤷